src/server/pegasus_server_impl.cpp

```diff
@@ -2634,6 +2635,10 @@ void pegasus_server_impl::update_usage_scenario(const std::map<std::string, std:
                            old_usage_scenario,
                            new_usage_scenario);
         }
+    } else {
+        // When an old db is opened and the conf is changed, the options related to usage scenario
```

Comment:
How about change 'the conf is changed' to 'the rocksDB related configs in server config.ini has been changed' ?